src/common/redis_module/ray_redis_module.cc

```diff
@@ -388,6 +388,46 @@ bool PublishObjectNotification(RedisModuleCtx *ctx,
   return true;
 }
 
+int TableAdd_RedisCommand(RedisModuleCtx *ctx,
+                          RedisModuleString **argv,
+                          int argc) {
+  if (argc != 3) {
+    return RedisModule_WrongArity(ctx);
+  }
+
+  RedisModuleString *id = argv[1];
+  RedisModuleString *data = argv[2];
+
+  // Set the keys in the table
+  RedisModuleKey *key;
+  key = OpenPrefixedKey(ctx, "T:", id, REDISMODULE_READ | REDISMODULE_WRITE);
+  RedisModule_StringSet(key, data);
+  RedisModule_CloseKey(key);
+
+  return RedisModule_ReplyWithSimpleString(ctx, "OK");
+}
+
+int TableLookup_RedisCommand(RedisModuleCtx *ctx,
+                             RedisModuleString **argv,
+                             int argc) {
+  if (argc != 2) {
+    return RedisModule_WrongArity(ctx);
+  }
+
+  RedisModuleString *id = argv[1];
+
+  RedisModuleKey *key;
+  key = OpenPrefixedKey(ctx, "T:", id, REDISMODULE_READ);
+  size_t len;
```

Comment:
In local scope I think it's a good practice to always initialize: `size_t len = 0;`.  I could be misremembering, but such a statement was an undefined behavior?

Reply:
Yeah, you are right, it is undefined behavior in the sense that if it is accessed the value is undefined (in many cases it will just be whatever was in the corresponding memory location on the stack). In this case it would be fine because `RedisModule_StringDMA` will initialize the memory location and not read before it, but I'm initializing it nonetheless :)